topic_dynamics/parsing.py

```diff
@@ -21,18 +22,8 @@
               "java": ["identifier", "type_identifier"],
               "python": ["identifier", "type_identifier"]}
 
-
-class DataForWriting:
-    def __init__(self, a, b):
-        self.a = a, self.b = b
-
-    def __repr__(self):
-        return str(self.a) + "," + str(self.b)
-
-    @classmethod
-    def parse(string):
-        a, b = map(int, string.split(","))
-        return DataForWriting(a, b)
+SliceLine = namedtuple("SliceLine", "date starting_index end_index")
```

Comment:
it should be reused

Reply:
Reused in modeling from here (since here it is logical for the first time)